generate/generate.go

```diff
@@ -32,7 +37,7 @@ func savePrivateKey(privateKey *rsa.PrivateKey) error {
 		Type:  "RSA PRIVATE KEY",
 		Bytes: PrivASN1,
 	})
-	ioutil.WriteFile("private.key", privBytes, 0644) // FIXME @antoniaklja shall we pass destination path here?
+	ioutil.WriteFile(PrivateKeyFile, privBytes, 0644) // FIXME @antoniaklja shall we pass destination path here?
```

Comment:
what dou you mean by "destination path"?

I would stick to generating it in the same dir for now and remove the fixme, it's not a bug or a debt IMO, it's just a potential feature.

Reply:
sure